modules/Link.js

```diff
@@ -94,7 +112,7 @@ class Link extends React.Component {
         className={isActive ?
           [ className, activeClassName ].join(' ').trim() : className
         }
-      />
+      >{children}</a>
```

Comment:
Instead of including the children inside of the `<a>` (which makes a nested set of a's, which is against spec and your browser will mess with), I would if/else between children or the `<a>` and use `cloneElement`. I would also require a single child so you can do that easily.